code/API_definitions/geofencing.yaml

```diff
@@ -43,7 +43,7 @@ info:
     
     Note: Additionally to these list, ``org.camaraproject.geofencing.v0.subscription-ends`` notification `type` is sent when the subscription ends. 
     This notification did not requires dedicated subscription. 
```

Comment:
Minor typo here. Should read: "This notification **does not require** dedicated subscription." 
Thank you.